styles/pinterest/catppuccin.user.css

```
@@ -112,10 +112,12 @@
     --color-background-tag-primary-default: @surface1;
     --color-background-button-disabled-default: darken(@mantle, 2%);
     --color-background-formfield-primary: @surface0;
+    --color-background-avatar-placeholder: @mantle;
     --color-border-container: @surface1;
     --color-border-default: @mantle;
     --color-border-popover-primary: @surface0;
     --color-border-popover-secondary: @text;
+    --color-border-focus: fade(@blue, 50%);
```

perhaps accent-color?
```suggestion
    --color-border-focus: fade(@accent-color, 50%);
```

---

Some userstyles use blue for links (like what the style guide says) and some others styles just use the accent color, which can be understandable if the website uses its brand color for links.
In this case I thought it would be more appropriate to use the blue color for focus borders like in the default (I would make it accent if it was red, which isn't the case here)